src/app/components/utils/chart/chart.component.html

```diff
@@ -9,4 +9,184 @@
       <button mat-button mat-raised-button color="primary" (click)="randomize()">Update</button>
     </div>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+ -->
+
+<script src="https://d3js.org/d3.v4.min.js"></script>
```

Comment:
Not recommended to import the scripts directly here, as it might affect other components rendered on the html